package.json

```diff
@@ -4,17 +4,20 @@
   "description": "[![Build Status](https://travis-ci.org/atlp-rwanda/space-bn-backend.svg?branch=develop)](https://travis-ci.org/atlp-rwanda/space-bn-backend)",
   "main": "index.js",
   "scripts": {
-    "migrate": "sequelize db:migrate",
+    "migrate": "sequelize db:migrate && npm run admin",
     "seed": "sequelize db:seed:all",
     "migrate:reset": "sequelize db:migrate:undo:all && npm run migrate",
-    "test": "cross-env NODE_ENV=test nyc --reporter=html --reporter=text mocha --require @babel/register ./src/tests/*.test.js --recursive ./src/tests/*.test.js --exit --timeout 20000",
+    "test": "cross-env NODE_ENV=test nyc --reporter=html --reporter=text mocha --require @babel/register ./src/tests/ --recursive  --exit --timeout 20000 ",
     "pretest": "cross-env NODE_ENV=test npm run migrate:reset",
     "lint": "eslint ./",
     "coveralls": "cat ./coverage/lcov.info | coveralls",
     "coverage": "nyc reporter --reporter=text --lcov | coveralls",
     "start:dev": "nodemon --exec babel-node src/index.js",
     "start": "nodemon --exec babel-node src/index.js",
```

Comment:
Help us to differentiate **start:dev** and **start** (i.e. On **start** command, we do not need nodemon).

Reply:
That has been there before, so I was not supposed to change that in my PR, therefore that would be a fix not implemented here